RFS/src/main/java/org/opensearch/migrations/bulkload/lucene/version_9/ZstdStoredFields814Format.java

```diff
@@ -0,0 +1,36 @@
+package org.opensearch.migrations.bulkload.lucene.version_9;
+
+import java.io.IOException;
+
+import shadow.lucene9.org.apache.lucene.codecs.StoredFieldsFormat;
+import shadow.lucene9.org.apache.lucene.codecs.StoredFieldsReader;
+import shadow.lucene9.org.apache.lucene.codecs.StoredFieldsWriter;
+import shadow.lucene9.org.apache.lucene.codecs.lucene90.compressing.Lucene90CompressingStoredFieldsFormat;
+import shadow.lucene9.org.apache.lucene.index.*;
+import shadow.lucene9.org.apache.lucene.store.Directory;
+import shadow.lucene9.org.apache.lucene.store.IOContext;
+
+public class ZstdStoredFields814Format extends StoredFieldsFormat {
+
+    public ZstdStoredFields814Format() {
+        System.out.println(">>> Loaded ZstdStoredFields814Format as a fallback stored fields decoder.");
+    }
+
+    @Override
+    public StoredFieldsReader fieldsReader(Directory directory, SegmentInfo si, FieldInfos fn, IOContext context) throws IOException {
+        System.out.println(">>> Attempting to decode stored fields using ZstdStoredFields814Format for segment: " + si.name);
+        // TODO: Replace with real reader implementation
+        return new Lucene90CompressingStoredFieldsFormat(
```

Comment:
I think that importing [custom-codecs](https://github.com/opensearch-project/custom-codecs/) implementation would be a better path that rolling our own implementation.  Looks like Lucene912CustomStoredFieldsFormat  might be the file we are looking for [link](https://github.com/opensearch-project/custom-codecs/blob/2.19.2.0/src/main/java/org/opensearch/index/codec/customcodecs/Lucene912CustomStoredFieldsFormat.java).

Here it is on maven, https://mvnrepository.com/artifact/org.opensearch.plugin/opensearch-custom-codecs

Since we are using lucene 9, make sure to use their 2.19.2 artifact

Reply:
@jugal-chauhan Why aren't we importing the custom codecs?